spells.yml

```diff
@@ -6885,6 +7313,28 @@ phantom_steed:
     - illusion
     - ritual
     - wizard
+phantom_warriors:
+  casting_time: 1 standard action
+  components: V
+  concentration: true
+  description: |-
+    Up to four illusory, Medium- or Small-sized phantom warriors appear in unoccupied spaces within range. Each creature's appearance and behavior is up to you. They appear real, visually and audibly, but are incorporeal.
```

Comment:
Is it actually correct to leave "-" attached to Medium here?

Reply:
It is